m4_sessions/1705-inning/UBI.md

```diff
@@ -18,7 +15,13 @@ Our mission is to “unlock human potential.” The software we write enables it
 solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs
 they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
 
-
+## Outline
+1. Into/ General UBI info - 10 min
```

Comment:
Use google slides if possible so this can be shared.  Also consider dropping a few resource in the channel and then adding your people before the session.